.config/dotnet-tools.json

```diff
@@ -15,7 +15,7 @@
       ]
     },
     "microsoft.dotnet.xharness.cli": {
-      "version": "1.0.0-prerelease.20265.8",
+      "version": "1.0.0-prerelease.20271.3",
```

Comment:
Should we update the `xharness` runners as well to match this version?

Reply:
@safern xharness runners are updated automatically, but not the tools as far as I understand (yet).